packages/flutter/test/widgets/flow_test.dart

```diff
@@ -2,16 +2,14 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// @dart = 2.8
-
 import 'package:flutter/rendering.dart';
 import 'package:flutter/widgets.dart';
 import 'package:flutter_test/flutter_test.dart';
 
 class TestFlowDelegate extends FlowDelegate {
   TestFlowDelegate({this.startOffset}) : super(repaint: startOffset);
 
-  final Animation<double> startOffset;
+  final Animation<double>? startOffset;
```

Comment:
maybe this should be required in the constructor instead of `?`. It's strange that in paintChildren we just assume its non-null but the constructor allows a null-value. Since the field is final, it can never change away from that null-value in its lifetime.